eng/scripts/smoketest/Get-Package-Version.ps1

```diff
@@ -0,0 +1,63 @@
+. (Join-Path $PSScriptRoot ../../common/scripts/common.ps1)
+$ErrorActionPreference = 'Continue'
+
+$NugetSource="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v2"
+$allPackages = Get-AllPkgProperties
+$trackTwoPackages = $allPackages.Where({ $_.IsNewSdk })
+
+$azureCorePkgInfo = $trackTwoPackages.Where({ $_.Name -eq "Azure.Core"})
+$azureCoreVer = [AzureEngSemanticVersion]::ParseVersionString($azureCorePkgInfo.Version)
+$azureCoreVer.IsPreRelease = $false
+$azureCoreVerBase = $azureCoreVer.ToString()
+
+$azureCorePkg = Find-Package -Name 'Azure.Core' `
+  -Source $NugetSource `
+  -AllowPrereleaseVersions `
+  -MinimumVersion "$azureCoreVerBase-alpha.202106" `
+  -MaximumVersion "$azureCoreVerBase-alphab" `
+  -Force
+
+Write-Host "Azure.Core Version: $($azureCorePkg.Version)"
+$azureCoreVerExtension = $azureCorePkg.Version.Replace($azureCoreVerBase, "")
+$azureCoreVerDateStr = $azureCoreVerExtension.split(".")[1]
+$azureCoreVerDate = [DateTime]::ParseExact($azureCoreVerDateStr, "yyyyMMdd", $null)
+$pkgMinVer = $azureCoreVerDate.addMonths(-1).ToString("yyyyMMdd")
```

Comment:
The code is pretty clear, but I think it's still worth a comment explaining what you're doing here.